.github/workflows/ci.yml

```
@@ -3,6 +3,9 @@ name: CI
 env:
   IMAGE_NAME: activemq-artemis-operator
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  OPM_VERSION: v1.54.0
+  OPERATOR_SDK_VERSION: v1.28.0
+  YQ_VERSION: v4.46.1
```

The prerequisites are already installed by the bundle and catalog targets
```suggestion
  OPM_VERSION: v1.54.0
  OPERATOR_SDK_VERSION: v1.28.0
  YQ_VERSION: v4.46.1
```